02-clients/07-new-client.md

```diff
@@ -611,64 +647,75 @@ message Transaction {
   }
 }
 ```
-Each **request message** is suffixed with `.Req`, and has a matching `.Res` (or `.ResPart`) to represent the server's response to that message.
+Each **request message** is suffixed with `.Req`, and has a matching `.Res` (or `.ResPart`) to represent the server's 
+response to that message.
 
-Now, there are two basic patterns to the communications; _single_ responses and _streamed_ responses, both of which are illustrated below.
+Now, there are two basic patterns to the communications; _single_ responses and _streamed_ responses, both of which 
+are illustrated below.
 
 ![Inside a Transaction Stream](../images/client-api/tx-stream.png)
```

Comment:
Oh hey. This is where the `response-structure.png` was suppose to go. `tx-stream` is a better name for the diagram though.